sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/_tests_/msfaa-part-time-process-response-integration.scheduler.e2e-spec.ts

```diff
@@ -157,6 +166,23 @@ describe(
       // Validate second confirmed record.
       expect(secondSignedMSFAA.dateSigned).toBe("2021-11-22");
       expect(secondSignedMSFAA.serviceProviderReceivedDate).toBe("2021-11-23");
+
+      // Get the notification using the student and notification message id. Since a fake student is created for every test run, it will ensure uniqueness.
+      // Expecting one notification for the cancelled MSFAA record.
+      const studentUserId = cancelledMSFAA.student.user.id;
+      const notificationMessageType = NotificationMessageType.MSFAACancellation;
+      const notificationsQuery = db.notification
+        .createQueryBuilder("notifications")
+        .select(["notifications.id", "notifications.dateSent"])
+        .innerJoin("notifications.user", "users")
```

Comment:
https://github.com/bcgov/SIMS/pull/2069/files#r1248264087

Reply:
https://github.com/bcgov/SIMS/pull/2069/files#r1248311140

Reply:
Can you try the below one?
```ts
db.notification.find({
  select: {
    id: true,
    dateSent: true,
  },
  where: {
    notificationMessage: {
      id: notificationMessageType,
    },
    user: {
      id: studentUserId,
    },
  },
});
```

Reply:
Thank You @andrewsignori-aot